build.gradle.kts

```diff
@@ -6,6 +6,7 @@ plugins {
     alias(libs.plugins.android.library) apply false
     alias(libs.plugins.kotlin.android) apply false
     alias(libs.plugins.ksp) apply false
+    alias(libs.plugins.kover) apply false
```

Comment:
se for colocar em todos eu acho mais facil colocar dentro dos plugins de library e application

Reply:
Eu tentei colocar mas quebrou, parece que nos plug-ins library e application, o kover não é reconhecido 😞 

Podemos meter um pair pra ti me ajudar  😀

Reply:
lendo agora o código esse ai não iria dar certo mesmo que é plugin e vai ficar no build gradle do projeto mesmo